static_map: "/c2g/static_map"
inferred_pose_t: "/car_pose"
